src/helpers/paginationHelper.js

```diff
@@ -0,0 +1,34 @@
+import errorResponse from './index';
+
+/**
+ * @description class representing pagination helper functions
+ * @class paginationHelper
+ */
+class paginationHelper {
+  /**
+   * @description - This method is responsible for returning numbeer of article per page
+   * @static
+   * @param {number} number - parameter sent to controller for get all article
+   * @returns {number} - object representing response message
+   * @memberof paginationHelper
+   */
+  static numberOfArticlePerPage(number) {
```

Comment:
Nice work @okoroemeka, looking at this function, it doesn't seem to have any logic inside it. I think it would be better to remove it.